src/components/ThemeToggle/ThemeToggle.module.scss

```diff
@@ -0,0 +1,22 @@
+@import 'styles/uswdsDependencies';
+@import 'styles/sfds/sfdsDependencies';
+
+.toggleButton {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  width: 132px;
+  height: 30px;
+  left: 0px;
+  top: 0px;
+  background: $theme-aurora-dark;
```

Comment:
Suggestion: Can this be changed to use an appropriate button var from the theme?

Reply:
Yes! Meant to do that, but forgot.

Reply:
Actually, looking back at the design it looks like the background color doesn't change across themes. Same for text color. I updated the text to make sure it stays black though, but maybe this is another thing to bring up when Mallory gets back?

Reply:
Now that you mention it I think the header is black in both modes. Still good to confirm with Mallory imo.